package.json

```diff
@@ -27,7 +27,7 @@
     "link-win": "node ./scripts/link-bin.js packages/amplify-cli/bin/amplify amplify-dev",
     "lint-check": "yarn eslint . --ext .js,.jsx,.ts,.tsx --max-warnings=1085",
     "lint-fix": "git diff --name-only --cached --diff-filter d | grep -E '\\.(js|jsx|ts|tsx)$' | xargs eslint --fix --quiet",
-    "lint-check-package-json": "yarn eslint --no-eslintrc --config .eslint.package.json.js '**/package.json'",
+    "lint-check-package-json": "yarn eslint --no-eslintrc --config .eslint.package.json.js '**/package.json' || echo 'The dependencies in `package.json` are not sorted correctly, please run yarn sort-packages to fix this issue.'",
```

Comment:
Is sorting the only reason this lint rule can fail?

Reply:
hmmmm, this is a good point. Let me take a look at the lint rules